components/centraldashboard/public/components/main-page.js

```diff
@@ -173,10 +173,12 @@ export class MainPage extends utilitiesMixin(PolymerElement) {
      * @param {Event} ev AJAX-response
      */
     _onHasWorkgroupResponse(ev) {
-        const {user, hasWorkgroup, hasAuth} = ev.detail.response;
+        const {user, hasAuth, namespaceCreation} = ev.detail.response;
```

Comment:
Let's not remove the `hasWorkgroup` logic (since that is the actual trigger for registration flow to work)

Reply:
Should be good now.